lib/commands/find.js

```diff
@@ -16,55 +35,39 @@ helpers.findElOrEls = async function (strategy, selector, mult, context = '') {
     throw new Error("Must provide a selector when finding elements");
   }
 
-  let params = {
+  this.params = {
     strategy,
     selector,
     context,
     multiple: mult
   };
 
-  let element;
   let doFind = async () => {
-    try {
-      element = await this.bootstrap.sendAction('find', params);
-    } catch (err) {
-      if (err.message && err.message.match(/An element could not be located/)) {
-        // we are fine with this, just indicate a retry
-        return false;
-      }
-      throw err;
-    }
-
-    // we want to return false if we want to potentially try again
-    if (mult) {
-      return element && element.length !== 0;
-    } else {
-      return !_.isNull(element);
-    }
+    return this.doFindElementOrEls();
```

Comment:
We've been enforcing `return await someAsyncCall();` so that we get proper translation of promises in the event that the promises come from different libs.